allennlp/data/semparse/worlds/nlvr_world.py

```diff
@@ -32,19 +34,26 @@ class Object:
         The dict for each object from the json file.
     """
     def __init__(self, attributes: JsonDict) -> None:
-        object_color = attributes["color"]
+        object_color = attributes["color"].lower()
         # The dataset has a hex code only for blue for some reason.
         if object_color.startswith("#"):
             self.color = "blue"
         else:
-            self.color = object_color.lower()
-        self.shape = attributes["type"].lower()
+            self.color = object_color
+        object_shape = attributes["type"].lower()
+        self.shape = object_shape
         self.x_loc = attributes["x_loc"]
         self.y_loc = attributes["y_loc"]
         self.size = attributes["size"]
 
     def __str__(self):
-        return "%s %s at (%d, %d)" % (self.color, self.shape, self.x_loc, self.y_loc)
+        if self.size == 10:
+            size = "small"
+        elif self.size == 20:
+            size = "medium"
+        else:
+            size = "big"
+        return "%s %s %s at (%d, %d)" % (size, self.color, self.shape, self.x_loc, self.y_loc)
```

Comment:
You could consider using the `f"{size} {self.color} ..."` syntax from python 3.6.

Reply:
Considered and changed. :)